inst/rmarkdown/templates/visc_report/resources/template.tex

```diff
@@ -121,15 +121,26 @@
   citecolor  = gray  % color of citations
 }
 
-% pandoc now requires a cslreferences environment to be defined in the latex template
-$if(csl-refs)$
 \newlength{\cslhangindent}
 \setlength{\cslhangindent}{1.5em}
-\newenvironment{cslreferences}%
-  {$if(csl-hanging-indent)$\setlength{\parindent}{0pt}%
-  \everypar{\setlength{\hangindent}{\cslhangindent}}\ignorespaces$endif$}%
-  {\par}
-$endif$
```

Comment:
Referencing this document: https://github.com/jgm/pandoc/blob/master/data/templates/default.latex

Looks like you need to add back in `$if(csl-refs)$` and `$endif$`. Then I will merge!

Reply:
Added in.....thanks for catching this